packages/matchbox/src/components/Layout/SectionTitle.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Text } from '../Text';
+
+function SectionTitle(props) {
+  const { children } = props;
+
+  return (
+    <Text as="h4" fontSize={['300', null, '400']} pb="300">
```

Comment:
I think i would make these `h2`, but then also accept and `as` prop so FE can override them